provider/pdns/pdns.go

```diff
@@ -316,12 +316,15 @@ func (p *PDNSProvider) ConvertEndpointsToZones(eps []*endpoint.Endpoint, changet
 				for _, t := range ep.Targets {
 					if ep.RecordType == "CNAME" || ep.RecordType == "ALIAS" {
 						t = provider.EnsureTrailingDot(t)
-						if t != zone.Name && !strings.HasSuffix(t, "."+zone.Name) {
-							RecordType_ = "ALIAS"
-						}
 					}
 					records = append(records, pgo.Record{Content: t})
 				}
+
+				if dnsname == zone.Name && ep.RecordType == "CNAME" {
+				    log.Debugf("Converting APEX record %s from CNAME to ALIAS", dnsname)
+					RecordType_ = "ALIAS"
```

Comment:
```suggestion
					log.Debugf("Converting APEX record %s from CNAME to ALIAS", dnsname)
					RecordType_ = "ALIAS"
```

Reply:
There is a linter issue, you can either gofmt -w -s on this file or (maybe) it will work with this suggestion